### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by armic**
Code cleanup for getResponse

### CODE REVIEW

File: getResponse.js [hunk 1]
```diff
@@ -1,69 +1,88 @@
...
+}
+
+// isWithinOperatingHours takes two strings representing opening
+// & closing times, each formatted as `hh:mm`, and returns true
+// if the current time is between those, false otherwise.
+function isWithinOperatingHours(openTime, closeTime) {
+  const currentDate = new Date();
+
+  // HACK: shifting current hours by 4 to account for time difference
+  const [currentHour, currentMinute] = [currentDate().getHours() - 4, currentDate.getMinutes()];
```
Comment (adelq) [inline getResponse.js]: I'm guessing this doesn't work with daylight savings?

**Review by adelq — COMMENTED** [review]
Much easier to understand what's going on here, just had a comment on the datetime handling.

File: getResponse.js (hunk 1 above)
Comment (armic) [reply getResponse.js]: @adelq Yeah it doesn't, probably the best solution will be to use momentjs to handle the dates

Comment (adelq) [reply getResponse.js]: Yup, that's what I was going to suggest. Otherwise I think LGTM

**Pull request merged**